scripts/bootstrap

```diff
@@ -15,13 +15,15 @@ Pull down secrets from S3.
 "
 }
 
-# TODO: potentially as a part of this issue: https://github.com/azavea/echo-locator/issues/376
-# after switching to a different bundler, we may want to copy and populate some env vars
 function pull_env() {
     pushd "${DIR}"
 
-    echo "Pulling .env from ${1}"
-    # aws s3 cp "s3://${1}/.env" ".env"
+    if aws s3 cp "s3://${ECHOLOCATOR_SETTINGS_BUCKET}/.env" ".env"; then
+        echo "Updated environment variables file from ${ECHOLOCATOR_SETTINGS_BUCKET}"
+    elif [[ ! -f ".env" ]]; then
+        cp .env.sample .env
+        echo "Updated environment variables file from .env.sample"
```

Comment:
Maybe make this error make it clear that it's ok if this fails and you want to do local development.

Reply:
Sounds good. How about

>"Updated environment variables file from .env.sample for local development. Please populate the fields in this file manually."

Reply:
Updated here: https://github.com/azavea/echo-locator/pull/500/commits/689495395de9c42c5a485d38b62819572c2a5eab